docs/design/coreclr/botr/xplat-minidump-generation.md

```diff
@@ -79,14 +79,27 @@ The createdump utility can also be run from the command line on arbitrary .NET C
 
 `sudo createdump <pid>`
 
-    createdump [options] pid
-    -f, --name - dump path and file name. The pid can be placed in the name with %d. The default is "/tmp/coredump.%d"
-    -n, --normal - create minidump (default).
-    -h, --withheap - create minidump with heap.
+    -f, --name - dump path and file name. The %p, %e, %h %t format characters are supported. The default is '/tmp/coredump.%p'
+    -n, --normal - create minidump.
+    -h, --withheap - create minidump with heap (default).
     -t, --triage - create triage minidump.
     -u, --full - create full core dump.
     -d, --diag - enable diagnostic messages.
 
+
+**Dump name formatting**
+
+As of .NET 5.0, the following subset of the core pattern (see [core](https://man7.org/linux/man-pages/man5/core.5.html)) dump name formatting is supported:
+
+    %%  A single % character.
+    %d  PID of dumped process (for backwards createdump compatibility).
+    %p  PID of dumped process.
+    %P  PID of dumped process.
+    %e  The process executable filename.
+    %E  The process executable filename.
```

Comment:
%E is documented in that man page to be the **path** of the executable:
```
 %e  The process or thread's comm value, which typically is the
               same as the executable filename (without path prefix,...
%E  Pathname of executable, with slashes ('/') replaced by
               exclamation marks ('!') (since Linux 3.0).
```
We should either enable %E to have the path, or block %E. 

In the context of developing .NET we often have several distinct dotnet exe's so I could see some value in %E but I don't have a strong opinon. It does seem we should block it now if we don't implement it now so we can do later.

Reply:
I'll just remove %E support because it is hard to get the full exe path on all the platforms.

Reply:
It is not that hard, we have all the code in corerun here, so you can reuse it:
https://github.com/dotnet/runtime/blob/2adf9da74c3dffffee5c425bfd54af9014967d02/src/coreclr/src/hosts/unixcoreruncommon/coreruncommon.cpp#L59

Reply:
This function would work great in-process but this is in createdump and I need the executable name/path for the remote process it is generating the dump.

Reply:
Ah, I am sorry, I've missed that.